tests/control_loop_test.py

```diff
@@ -36,13 +40,22 @@ def setup_streams(input_streams):
 
     @frequency(1)
     def publish_msg(self):
+        if self._cnt == MAXIMUM_COUNT:
+            sys.exit()
```

Comment:
As above. I think we need a mechanism to exit from operators.
How can you distinguish between a correct or a failed execution when you invoke `sys.exit`?